insuree/gql_mutations.py

```diff
@@ -57,6 +57,12 @@ class InsureeBase:
     status = graphene.String(required=False)
     status_reason = graphene.String(required=False)
     status_date = graphene.Date(required=False)
+    poligamous = graphene.Boolean(required=False)
```

Comment:
There are several attributes repeating in Family and Insuree, and I think this should not happen: poligamous, coordinates, preferred_payment_method, income_level_id, professional_situation, bank_coordinates.

Please fix spelling meeting polygamous.